Gemfile

```diff
@@ -70,6 +70,8 @@ group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'bullet'
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails'
+  gem 'faker'
 end
 
 group :development do
```

Comment:
- [ ] Kindly add `gem 'ffi'` to your Gemfile to enable all users with different OS to run your tests successfully because I got an error that prompt me to add the mentioned gem before I was able to run your tests successfully.